README.md

```diff
@@ -1,11 +1,11 @@
 # Efficient Bound of Lipschitz Constant for Convolutional Layers by Gram Iteration
```

Comment:
This is the title of ICML article.
I think it would be good to find a title that encompasses both articles.
Suggestion: "Spectral norm and Lipschitz bound for Convolutional Layers"